package-overrides/github/vitalets/angular-xeditable@0.1.8.json

```diff
@@ -1,3 +1,17 @@
 {
-  "main": "dist/js/xeditable.min"
-}
\ No newline at end of file
+  "directories": {
+    "lib": "dist"
```

Comment:
Let's leave this out here to maintain the structure since the override is already in use.